mystery_word.py

```diff
@@ -0,0 +1,72 @@
+def create_list():
+    """Takes a list of strings from a file and creates a list of words
+        striped of all spaces and seperated by space. Returns one word from
+        the list of strings"""
+
+
+    with open("/usr/share/dict/words") as file:
+
+        import random
+
+        file = file.readlines()
+        file = [word.strip() for word in file]
+        random_word = random.choice(file).lower()
+
+    return random_word
+random_word = create_list()
+
+def find_length(random_word):
+    """Tells you how many letters are in the word and prints out the length represented by _"""
+    displayed_word = ("_ " * (len(random_word)))
+    print("There are {} letters in this word".format(len(random_word)))
+    print(displayed_word)
+    return(displayed_word)
+find_length(random_word)
+
+
+turns_left = 8
+wrong_letters = []
+guesses = []
+
+
+while turns_left > 0:
+    guess = input("Guess a letter: ").lower()
+    guess = guess.lower()
+
+    win =0  #I am constantly trying to add 1 to the win. The only way the win
+            #has a value of 0 is if it has letters in all the spaces where the
+            #_ should be.
+
+    if len(guess) != 1:
+        print("Guess one letter at a time")
+        continue
+
+    if guess not in random_word:
+        win += 1
+
+        if guess not in wrong_letters:
+            wrong_letters += guess
+        turns_left -= 1
+        print("Sorry, that letter is not in the word")
+        if turns_left == 0:
+            print("YOU LOSE!!!!")
+        print("\n You have {} more guesses.".format(turns_left))
+        print("\n Guessed wrong letters: {}".format(wrong_letters))
+
+
+    if guess in random_word:
+        for char in random_word:
+            guesses += guess
+            if char in guesses:
+                print(char , end = "")
+            else:
+                print("_ ", end = "")
+                win += 1
+        print("\n")
```

Comment:
This prints strangely. I would make sure you leave a space after guessed letters as well.